docs/glassfish/parent/pom.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
why is the parent pom in a specific directory instead of being docs/glassfish/pom.xml ?